Sources/SourceKitLSP/Swift/SemanticTokens.swift

```diff
@@ -58,7 +58,10 @@ extension SwiftLanguageService {
       if let range = range.flatMap({ snapshot.byteSourceRange(of: $0) }) {
         range
       } else {
-        ByteSourceRange(offset: 0, length: await tree.totalLength.utf8Length)
+        Range<AbsolutePosition>(
+          position: AbsolutePosition(utf8Offset: 0),
+          length: SourceLength(utf8Length: await tree.totalLength.utf8Length)
+        )
```

Comment:
I think this could just be `tree.range`

```suggestion
        await tree.range
```